spec/datadog/tracing/distributed/datadog_spec.rb

```diff
@@ -103,6 +109,8 @@
         end
 
         context 'nil' do
+          include_context 'with 128-bit trace_id generation disabled'
```

Comment:
I would prefer making examples more explicit by repeating 
```
before do
  Datadog.configure { |c| c.tracing.trace_id_128_bit_generation_enabled = false }
end
```
Instead of adding an indirection
```
include_context 'with 128-bit trace_id generation disabled'
```

Reply:
I agree here, given `c.tracing.trace_id_128_bit_generation_enabled = false` reads pretty much the same as `with 128-bit trace_id generation disabled`.

Reply:
I disagree. I think the shared_context is a bit clearer in communicating intentions with the tests as well as removes some clutter. But since the actual context is so small I don't think it's the end of the world to revert this.